### PR TITLE
springer card settings

### PR DESCRIPTION
**Pull request by Heydon**
* Springer `global-card` settings
* Support for `border` in all cards
* Support for `box-shadow` in all cards

### CODE REVIEW

File: toolkits/global/packages/global-card/demo/main.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
-@import '../node_modules/@springernature/brand-context/default/scss/core.scss';
-@import '../node_modules/@springernature/brand-context/default/scss/enhanced.scss';
+@import '../node_modules/@springernature/brand-context/springer/scss/core.scss';
```
Comment (tulipdexter) [inline toolkits/global/packages/global-card/demo/main.scss:1]: Out of curiosity, will the 'demo' for all components be the Springer brand, or can it vary?

File: toolkits/global/packages/global-card/scss/10-settings/springer.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+// Default setting
+@import 'default';
+
+// Springer overrides
+$card--padding-x: 0;
+$card--dark-background: #00285a;
+$card--dark-color: #fff;
+$card--title-font-weight: normal;
+$card--title-heading: h3;
```
Comment (tulipdexter) [inline toolkits/global/packages/global-card/scss/10-settings/springer.scss:14]: How does this `h3` work?

Comment (Heydon) [reply toolkits/global/packages/global-card/scss/10-settings/springer.scss:14]: @tulipdexter I'm not sure, I didn't write that.

Comment (Heydon) [reply toolkits/global/packages/global-card/scss/10-settings/springer.scss:14]: Looks like it's passed to a mixin:

```
@include u-heading($card--title-heading);
```

**Comment by Heydon** [comment]
@tulipdexter Whoops, I forgot to push my second commit with the versioning.

You can adjust the demo to use whichever brand, but that was a good spot: it should really be default by, erm, default. Thanks!

File: toolkits/global/packages/global-card/scss/10-settings/springer.scss (hunk 2 above)
Comment (tulipdexter) [reply toolkits/global/packages/global-card/scss/10-settings/springer.scss:14]: Oh I see! Wow that really threw me. Thanks 👍

File: toolkits/global/packages/global-card/HISTORY.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # History
 
+## 5.0.0 (2021-11-08)
```
Comment (tulipdexter) [inline toolkits/global/packages/global-card/HISTORY.md:3]: Just based on the breaking change below, should it be structured like this?

```
* BREAKING:
    * bla bla
```

I don't know if that's a _thing_ or just personal preferences. Just caught my eye.

**Review by tulipdexter — APPROVED** [review]

File: toolkits/global/packages/global-card/HISTORY.md (hunk 3 above)
Comment (Heydon) [reply toolkits/global/packages/global-card/HISTORY.md:3]: @tulipdexter You're right, it is like that normally.

**Pull request merged**